### PR TITLE
Fixed issue with "StudyingForYears" application status

### PR DESCRIPTION
**Pull request by olsavchenk**

### CODE REVIEW

**Comment by olsavchenk** [comment]
Some of the changes are not connected to my issue I've found them during my investigation and I decided to fix them. In InstitutionHierarchyServiceTest there was unused mock and in ChildControllerTests there were extra tabs

File: OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChildServiceTests.cs [hunk 1]
```diff
@@ -1,178 +1,286 @@
...
+            parentRepositoryMock.Object,
+            socialGroupRepositoryMock.Object,
+            loggerMock.Object,
+            mapperMock.Object,
+            applicationRepositoryMock.Object,
+            parentConfigMock.Object);
+    }
+
+    [Test]
+    public async Task GetApprovedByWorkshopId_ExistingChildren_ShouldReturnChildren()
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChildServiceTests.cs:63]: (Test logic) It looks that this test would never be "red", as it checks something, but not you code changes. 

Try to remove from your `ChildService` added piece of code `|| p.Status == ApplicationStatus.StudyingForYears` and this test will not fail.

**Review by VadymLevkovskyi — CHANGES_REQUESTED** [review]
Please improve your tests

File: OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChildServiceTests.cs (hunk 1 above)
Comment (olsavchenk) [reply OutOfSchool/OutOfSchool.WebApi.Tests/Services/ChildServiceTests.cs:63]: Hopefully, I have fixed the tests, and now they will fail when the filter in ChildService is changed

**Review by VadymLevkovskyi — APPROVED** [review]

**Pull request merged**